projects/1.19/assets/dimensional-pockets-ii/dimensionalpocketsii/lang/zh_cn.json

```
@@ -0,0 +1,1072 @@
+	"block.dimensionalpocketsii.block_wall_smithing_table": "锻造台模块",
+	"block.dimensionalpocketsii.block_wall_furnace": "熔炉模块",
+	"block.dimensionalpocketsii.block_wall_generator": "发电机模块",
+	"block.dimensionalpocketsii.block_wall_blast_furnace": "高炉模块",
+	"block.dimensionalpocketsii.block_wall_energy_display": "能量显示模块",
+	"block.dimensionalpocketsii.block_wall_fluid_display": "流体显示模块",
+	"block.dimensionalpocketsii.block_wall_armour_workbench": "盔甲工作台模块",
+	"block.dimensionalpocketsii.block_wall_upgrade_station": "升级站模块",
+	"block.dimensionalpocketsii.block_wall_anvil": "铁砧模块",
+	"block.dimensionalpocketsii.block_wall_creative_energy": "创造模式能量模块",
+	"block.dimensionalpocketsii.block_wall_creative_fluid": "创造模式流体模块",
+		
+	"dimensionalpocketsii.modid": "次元口袋II",
+	"dimensionalpocketsii.version": "[5.1.3.0]",
+	
+	"dimensionalpocketsii.system_message.backup": "已创建[DimPockets]的备份",
+	
+	"dimensionalpocketsii.command.dimshift.usage": "/dimshift <目标维度>（使用英语单词，例如：overworld）",
+	"dimensionalpocketsii.command.dimshift.error.exists": "选定的维度不存在",
+	"dimensionalpocketsii.command.dimshift.error.cheats": "你需要开启作弊功能来使用该命令",
+	"dimensionalpocketsii.command.dimshift.error.creative": "你需要处于创造模式以使用该命令",
+	"dimensionalpocketsii.command.dimshift.error.same": "你已经位于该维度内了",
+	
+	"dimensionalpocketsii.command.dimhelp.header": "--- 展示次元口袋的帮助 ---",
+	"dimensionalpocketsii.command.dimhelp.usage": "/dimhelp [页数]",
+	
+	"dimensionalpocketsii.command.recoverpocket.usage": "/recoverpocket <区块x坐标>，<区块z坐标>（选定口袋内需要恢复的区块的位置）",
+	"dimensionalpocketsii.command.recoverpocket.usage_": "/recoverpocket length（这将显示在注册表中存在多少口袋）",
+	"dimensionalpocketsii.command.recoverpocket.error.exists": "在注册表内选定的口袋不存在",
+	"dimensionalpocketsii.command.recoverpocket.error.cheats": "你需要开启作弊功能来使用该命令",
+	"dimensionalpocketsii.command.recoverpocket.error.not_owner": "你不能移除不属于你的次元口袋",
+	"dimensionalpocketsii.command.recoverpocket.error.creative": "你需要处于创造模式来使用该命令",
+	"dimensionalpocketsii.command.recoverpocket.error.no_space": "你需要至少一个空的物品栏槽位来移除口袋",
+	
+	"dimensionalpocketsii.command.recoverpocket.success.single": "已成功移除你的口袋",
+	"dimensionalpocketsii.command.recoverpocket.success.single_no_owner": "已成功移除无所有者的口袋",
+	
+	"dimensionalpocketsii.command.recoverpocket.op.usage": "/recoverpocketop <区块x坐标>，<区块z坐标>（选定口袋内需要恢复的区块的位置）",
+	"dimensionalpocketsii.command.recoverpocket.op.error.exists": "在注册表内选定的口袋不存在",
+	"dimensionalpocketsii.command.recoverpocket.op.error.permission": "你需要以服务器管理员（OP）的身份来使用该命令",
+	"dimensionalpocketsii.command.recoverpocket.op.error.creative": "你需要处于创造模式来使用该命令",
+	
+	"dimensionalpocketsii.command.setspawn.usage": "/setspawn（设置你进入到你的口袋后在其内部的出生点）",
+	"dimensionalpocketsii.command.setspawn.success.pre": "成功将口袋出生点设置为：[",
+	"dimensionalpocketsii.command.setspawn.success.suff": "]",
+	"dimensionalpocketsii.command.setspawn.error.dimension": "你需要在口袋内使用该命令",
+	
+	"dimensionalpocketsii.command.transfer.success.old": "成功将口袋转让给：",
+	"dimensionalpocketsii.command.transfer.success.new": "已将他们的口袋转让给你",
```

```suggestion
	"dimensionalpocketsii.command.transfer.success.new": "已将对方的口袋转让给你",
```
they有第三人称中性单数的用法，算是人版本的it（）